h2o-core/src/main/java/water/rapids/ast/prims/advmath/AstCorrelation.java

```diff
@@ -60,6 +57,7 @@ public Val apply(Env env, Env.StackHelp stk, AstRoot asts[]) {
       default:
         throw new IllegalArgumentException("unknown use mode: " + use);
     }
+
     return fry.numRows() == 1 ? scalar(frx, fry, mode) : array(frx, fry, mode);
```

Comment:
Not a single column. This will return the correlation of one row of data with another row, which is a scalar. We do the same in our covariance task.